### ISSUE TITLE
Extra comma at the end of institutions

### ISSUE THREAD

**Issue by scottaubrey** [elifesciences/enhanced-preprints-client#160]
# Description

example: https://staging--epp.elifesciences.org/reviewed-preprints/81535

![Image](https://user-images.githubusercontent.com/375812/196392732-4f432318-93b8-4014-bae4-58ea317d77ee.png)
![Image](https://user-images.githubusercontent.com/375812/196392731-522d4c17-f964-4751-9c64-c571438c1206.png)

**Comment by nlisgo** [comment]
What is the expected behaviour?

Current behaviour is that the following style is applied to the expansion link:

```
::before {
  content: ", ";
}
```

Do we ever need that style on the expansion link?

When we don't need a comma we will need to introduce a space or padding.

A PR is in play that just introduces the comma when the show more link is present and a space when show less. I assume (perhaps incorrectly) that the same behaviour is expected for the author list.

https://github.com/elifesciences/enhanced-preprints-client/pull/159

**Comment by chris-huggins** [comment]
Tl;dr - it's fine.

I had intended no comma at the end when institutions fit on the page, so I think it looks fine in staging. 
e.g. in staging:

![image](https://user-images.githubusercontent.com/41907215/196660650-65d398bb-12aa-4bfc-aeb5-48fc0ad0b589.png)


When they are collapsed (i.e. some are hidden and we introduce "show [x] more" I had intended ellipsis before the "show [x] more". We're close so I'm happy with it as is but something to improve in future.

Design:
![image](https://user-images.githubusercontent.com/41907215/196660587-e703b721-fc84-4a55-93d6-1bf64ce64505.png)

(Although I think "show" is better than  "+")

Staging:
![image](https://user-images.githubusercontent.com/41907215/196660700-4eee3ca6-e618-4ae7-98c9-df0cc90b6f48.png)

**Comment by QueenKraken** [comment]
Reviewed and approved. thank you @nlisgo

**Comment by nlisgo** [comment]
Brought this back in as the PR was already in play since before the launch. Just needs a review.
